package.json

```diff
@@ -5,13 +5,24 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "setup": "yarn && cd front-end/ && yarn && yarn build && cd .. && node app.js"
+    "setup": "yarn && cd front-end/ && yarn && yarn build && cd .. && node app.js",
+    "lint": "./node_modules/.bin/eslint '*.js' --fix && ./node_modules/.bin/eslint './front-end/src/**/*.js' --fix"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "body-parser": "^1.18.3",
-    "cors": "^2.8.4",
     "express": "^4.16.4"
+  },
+  "devDependencies": {
+    "eslint": "5.6.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-react": "^7.11.1",
+    "husky": "^1.1.3"
```

Comment:
Husky is added, but I don't see it being used. Maybe worth implementing it, or adding it later? :)

Reply:
I added it but eslint has a lot of errors, so for now there is no pre commit hook! I forgot to remove it O:)